src/app/account/invitation/[inviteId]/signup/page.tsx

```diff
@@ -99,15 +98,29 @@ const SetupAccountForm: FC<InviteData> = ({ inviteId, email, orgName }) => {
 
     return (
         <form onSubmit={form.onSubmit((values) => createAccount(values))}>
-            <Flex direction="column" gap="lg" maw={500} mx="auto">
+            <Flex direction="column" gap="lg" maw={500} mx="auto" pb="xxl">
                 <Text size="md">
                     You&apos;ve been invited to join {orgName}. Please fill out the details below to create your
                     account.
                 </Text>
-                <TextInput label="Email" value={email} disabled />
+                <TextInput
+                    label="Email"
+                    radius="sm"
+                    value={email}
+                    disabled
+                    c="charcoal.9"
+                    styles={{
+                        input: {
+                            backgroundColor: theme.colors.charcoal[1],
+                            borderColor: theme.colors.charcoal[1],
+                            color: theme.colors.charcoal[9],
+                        },
+                    }}
+                />
 
-                <Group grow gap="md">
+                <Flex direction="row" gap="xl">
                     <TextInput
```

Comment:
Hey @jbwilson8 the TextInputs will fill their parent Flex container if their `flex-grow` is set to an identical value for both, typically `1`

```suggestion
                    <TextInput
                        flex="1"
```

https://developer.mozilla.org/en-US/docs/Web/CSS/flex